docs/circuit_cutting/tutorials/gate_cutting_to_reduce_swap_overhead.ipynb

```diff
@@ -0,0 +1,301 @@
+      "text/plain": [
+       "<Figure size 1634.66x294.311 with 1 Axes>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qpd_circuit = decompose_gates(circuit, [8, 20, 32])\n",
+    "qpd_circuit.draw(\"mpl\", scale=0.8)"
+   ]
+  },
+  {
```

Comment:
It'd be quite nice, right here, to pass `qpd_circuit` through the transpiler, at least to the stage that a layout is determined.  For this to work as expected, though, the circuit will need to be decomposed all the way to `SingleQubitQPDGate`s.  Otherwise, the routing layer will want to add a bunch of swaps to get the two cut qubits on adjacent qubits :joy_cat:.

I don't know that this is a pressing priority, but it would be very nice for this demonstration.  The motivation at the top of this tutorial shows an example where transpilation results in swap overhead.  For symmetry, it'd be nice to see the layout and routing with _reduced_ swap overhead.

Reply:
In tutorial 2, I have included a cell showing a specific decomposition of the QPD circuit and how its depth is less than half the depth of the original circuit after they are both transpiled.